scripts/run_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Would it be easier to write this whole thing as a Python script, given the arg parsing at the beginning? I wont block on that, but `argparse` is generally cleaner for this kind of stuff IMO.

Reply:
It'd clean up the argparse, but there'd be a boatload of shelling out and handling STDOUT/STDERR, so it'd probably be net neutral.

I went full Python on the version bump because it seemed silly to not be parsing the version string with the code that parses version strings.